{{cookiecutter.repo_name}}/tox.ini

```diff
@@ -58,14 +57,9 @@ deps =
     -r{toxinidir}/requirements/test.txt
 commands =
     touch tests/__init__.py
-    pylint {{ cookiecutter.app_name }}
-    pylint tests
-    pylint test_utils
-    pylint --py3k {{ cookiecutter.app_name }}
-    pylint --py3k tests
-    pylint --py3k test_utils
+    pylint {{ cookiecutter.app_name }} tests test_utils
+    pylint --py3k {{ cookiecutter.app_name }} tests test_utils
```

Comment:
Suppose this pull request gets merged? https://github.com/PyCQA/pylint/pull/1449. Internally it's clear they intend to support multiple arguments: https://github.com/PyCQA/pylint/blob/7cb3ffddfd96f5e099ca697f6b1e30e727544627/pylint/lint.py#L712